packages/cmf/src/sagas/http.js

```diff
@@ -11,6 +10,12 @@ import {
 	testHTTPCode,
 } from '../middlewares/http/constants';
 
+let defaultConfig = {};
+
+export function handleCSRFToken(config) {
+	return mergeCSRFToken(config)(config);
```

Comment:
This feels so weird :P
So if I'm not wrong you do this because your `CSRFTokenHeaderKey` is in your http config right ?